readthedocs/builds/forms.py

```diff
@@ -20,7 +20,7 @@ class Meta(object):
             'largest',
         )
 
-    def __init__(self, instance=None, *args, **kwargs):
+    def __init__(self, instance=None, *args, **kwargs):  # noqa
```

Comment:
I'm wondering if we should add `keyword-arg-before-vararg` to `prospector.yml`. Perhaps it's better to just add a few `noqa`s and not introduce the bug any further (as you did).

Reply:
I like prospector informing about this, but there are some particular cases (like this one) that we want to make an exception. I decided to add `noqa` instead of `pylint: disable=` because the line gets to long and then pep8 start failing, so I need to also disable that one and it ends ugly.